packages/common/src/downloads/_internal/canUseExportImageFlow.ts

```diff
@@ -7,5 +7,9 @@ import { IHubEditableContent } from "../../core/types/IHubEditableContent";
  * @returns whether the export image flow can be used
  */
 export function canUseExportImageFlow(entity: IHubEditableContent): boolean {
-  return entity.type === "Image Service";
+  const { type, typeKeywords = [] } = entity;
+  // Tiled Imagery services cannot be downloaded. This typeKeyword check
+  // is one way to distinguish between tiled and non-tiled imagery services.
+  // TODO: Consider checking item.url instead so reference items are also excluded.
```

Comment:
highlighting for visibility

Reply:
Also encapsulated in https://devtopia.esri.com/dc/hub/issues/10901